jujugui/static/gui/src/app/models/bundle.js

```diff
@@ -214,8 +214,11 @@ YUI.add('juju-bundle-models', function(Y) {
 
           @method getter
          */
-        getter: function() {
-          return this.get('promulgated');
+        getter: function(val) {
+          var promulgated = this.get('promulgated');
```

Comment:
I'm not completely sure if we still need this promulgated flag as the new data flow uses `is_approved` but it might be required for backwards compatibility.